src/jquery.sidebar.js

```diff
@@ -114,7 +112,15 @@
          * */
         self.on("sidebar:close", function(callback) {
             var properties = {};
-            properties[settings.side] = settings.range[0];
+            if (settings.side == "left" || settings.side == "right")
```

Comment:
Pull requests are automagically updated when you push on the branch from where you created the pull request.